shell/content/docs/en-us/whats-new.md

```diff
@@ -2,6 +2,33 @@
 title: What's New in 2.6
 ---
 
+
+## New in 2.6.7
+
+### Amazon Billing in the Marketplace
+
+- Now you can purchase support for Rancher through the AWS marketplace 
```

Comment:
Nit, but I think we want to match capitalization of official feature name
```suggestion
- Now you can purchase support for Rancher through the AWS Marketplace 
```

Reply:
```suggestion
- You can now purchase support for Rancher directly through the AWS Marketplace:  https://rancher.com/docs/rancher/v2.6/en/installation/cloud-marketplace/aws/
```

That link will be dead until docs are merged, you can see it here: https://staging.rancher.com/docs/rancher/v2.6/en/installation/cloud-marketplace/aws/

Maybe @wjimenez5271 has the link to the actual csp-adapter listing.

Reply:
Actually we shouldn't do that since there are different listings for diff countries. I like text above. Reposting: 

```
You can now purchase support for Rancher directly through the AWS Marketplace:  
https://rancher.com/docs/rancher/v2.6/en/installation/cloud-marketplace/aws/
```